src/routes/devices.ts

```diff
@@ -705,16 +707,14 @@ const upsertGatewayDownload = async (
 	} else {
 		await resinApi.patch({
 			resource: 'gateway_download',
+			id: {
+				is_downloaded_by__device: deviceId,
+				image: imageId,
```

Comment:
```suggestion
				image: imageId,
				is_downloaded_by__device: deviceId,
```
Reorder to match the sbvr